examples/regression/2-advanced-analysis/plot_both_uncertainties.py

```diff
@@ -86,20 +89,52 @@ def get_1d_data_with_normal_distrib(
         ("linear", LinearRegression()),
     ]
 )
+polyn_model_quant = Pipeline(
+    [
+        ("poly", PolynomialFeatures(degree=degree_polyn)),
+        ("linear", QuantileRegressor(
+            alpha=1e-9,
```

Comment:
`alpha=0` to be consistent with `LinearRegression`.

Reply:
done